docs/dev/basics.rst

```diff
@@ -37,6 +37,9 @@ rerun
 `make update` also applies the patches that can be found in the directories found in
 `patches`; the resulting branch will be called `patched`, while the commit specified in `modules`
 can be referred to by the branch `base`.
+To separately call `make update` was choosen, to let you decide when to update and reset the external
+repos. This will prevent potential loss of commits during local development amd allow building Gluon
+without the need of having an internet connection.
```

Comment:
Not really, given that `make download` is a thing.

Reply:
I assume you are referencing to OpenWrts make download. But that's out of Gluons Makefile ...

Anyway I updated the sentence, as I found a typo.